src/generators/project/index.js

```diff
@@ -64,12 +65,11 @@ Shorthand examples (assuming xyz is your project name):
             'Press ctrl+c to exit and rename project directory.'
         );
 
-      if (err) {
```

Comment:
Has the removal of these braces come from a repo specific code style config (prettier?) or from your own preferences? I prefer to include braces, but the primary thing is that it's consistent across contributors.

Reply:
This came from me. Prettier respects braces if they're in there I believe. I still haven't decided what I prefer, but in this case saw that they were unnecessary so removed them to save a few characters. Yes agree code style consistency is good! Let's include guidelines in our docs hub when implemented.